src/store/store.interface.ts

```
@@ -0,0 +1,11 @@
+export class StoreAdapterBase {
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)